src/springloadAnalytics.ts

```diff
@@ -72,7 +72,7 @@ export const init = ({
   options = { ...options, ...overrideOptions };
   analyInstance = Analytics({
     app: 'springload-analytics',
-    plugins: trackerPlugins,
+    plugins: trackerPlugins.filter(Boolean),
```

Comment:
Where dose this `Boolean` come from?

Reply:
This is for filtering out any plugins in the list that return `undefined`. So with the other change we can add in the `Analytics.init` config:
```
trackerPlugins: [
    Analytics.performancePlugin(),
  ],
```
When the component is Server Side Rendered the `Analytics.performancePlugin()` returns undefined as it won't load the `perfume` library.

The `Analytics` library that takes the `trackerPlugins` assumes that all of the items in that array will be of the type `plugins` so breaks if we pass undefined through in the array.